web/templates/beatmap.html

```diff
@@ -100,10 +100,10 @@
                     {{ $.T "Download" }}
                   </a>
                   <a
-                    href="https://kitsu.moe/d/{{ .Beatmapset.ID }}"
+                    href="https://chimu.moe/d/{{ .Beatmapset.ID }}"
```

Comment:
can we not make this a config option?

Reply:
resolved in 216c6522663e460a0dae9d3dcac4c87402068c5a